File: src/main/java/de/maggiwuerze/xdccloader/events/server/ServerUpdateEvent.java
package de.maggiwuerze.xdccloader.events.server;

import de.maggiwuerze.xdccloader.events.EntityUpdateEvent;
import de.maggiwuerze.xdccloader.model.entity.Server;

public class ServerUpdateEvent extends EntityUpdateEvent<Server> {

	public ServerUpdateEvent(Object source, Server server) {
		super(source, server);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/server/ServerUpdateEvent.kt
package de.maggiwuerze.xdccwebloader.events.server

import de.maggiwuerze.xdccwebloader.events.EntityUpdateEvent
import de.maggiwuerze.xdccwebloader.model.entity.Server

class ServerUpdateEvent(source: Any, server: Server) : EntityUpdateEvent<Server?>(source, server)